omod/src/main/webapp/resources/scripts/jquery-ui/js/openmrsSearch.js

```diff
@@ -586,11 +581,30 @@ function OpenmrsSearch(div, showIncludeVoided, searchHandler, selectionHandler,
                 else
                     self._table.numberOfPages = Math.floor(initialRowCount/self._table.fnSettings()._iDisplayLength)+1;
 
-            } else if(self.options.searchPhrase || self.options.doSearchWhenEmpty) {
-                if (self.options.searchPhrase == null) {
-                    self.options.searchPhrase = "";
+            } else{ 
```

Comment:
That was an oversight when I redid that section I have added a new commit to fix this.